src/extension/commandPaletteHandler.ts

```diff
@@ -226,8 +230,7 @@ export class CommandPaletteHandler {
             });
             if (!CommandPaletteHandler.elementInspector.pid) {
                 CommandPaletteHandler.elementInspector = null;
-                this.logger.error("React devtools is not installed. Run `npm install -g react-devtools` command in your terminal to install it.");
```

Comment:
Is it okay, that this line is deleted?

Reply:
Yes because it will now be handled with command rejection, that's how it would look after this change
![image](https://user-images.githubusercontent.com/32372875/50220336-23ff5780-03a3-11e9-9cb2-eff945c8b415.png)